docs/additional-resources/blue-green-migration.md

```diff
@@ -30,12 +68,17 @@ Some key steps, excluding automated tests, include:
 - Delete the source's DynamoDB table in `east` and `west` regions.
 - Toggle the deploy table's values for `migrate` and `source-table-version`.
 
-
 ## Manual Migration Steps
 
 The application kicks off a migration automatically if it detects migrations that need to be run in the codebase that haven't yet been run upon that environment. In order to force a migration, perform the following manual steps. You might do this if you were doing a package update that might impact the migration and wanted to test it fully.
 
-1. Remove a migration script item from the environment's `efcms-<ENV>-deploy` DynamoDB table that is expected to migrate, from the `migrationsToRun.js` file. For example, if `migrationsToRun.js` includes a script called `0001-test-script.js`, delete that item from the DynamoDB table.
-2. Kick off a CircleCI workflow, which will follow the steps above for Automated Migration Steps.
+1. Go into the environment's deploy table `efcms-<ENV>-deploy`
+2. Change the `migrate` flag to `true`
```

Comment:
will this successfully work since `is-migration-needed` in [setup-for-blue-green-migration.sh](https://github.com/ustaxcourt/ef-cms/blob/staging/setup-for-blue-green-migration.sh#L15) will return `SKIP_MIGRATION` true if that environment has already run all the expected migration scripts?

Reply:
i'm probably missing something here

Reply:
based on looking at elasticsearch in the AWS console, the `test` domains have no indexing rate despite the CI workflow saying that it's on the reindexing step. that leads me to believe that the old documentation is correct and in order to trigger a full migration and subsequent reindex, one of the expected migration scripts must be deleted from the environment. 🤔

Reply:
I might be missing something, but wouldn't it be faster to delete one of the migration scripts rather than running all migrations in the `migrationsToRun` file which seems to be what would happen by toggling the `migrate` and `destination-table-version` flags? Or, is the intention here that we would want to document how to force a *full* migration, aka all the migration scripts in the `migrationsToRun` file?

Reply:
@rosiehsmith -- great question. Maybe it would be easier, maybe not. I think it might be a matter of preference.

Another way to think about it is if you want to force a migration, but not actually have the data get altered, you could use the steps that I'm adding into the documentation. It's conceivable that a migration should not be run more than once (I think this has been the case in the past). 

I think I'm going to compromise and keep both steps in the documentation since they both work. Thanks for the questions. 🙌

Reply:
That makes sense. Thanks for the follow up!